### ISSUE TITLE
for verification also show address on map

### ISSUE THREAD

**Issue by prestonp** [open-austin/permit-server#57]
The texas geolocation uses fuzzy searching, so it would be good to show a map with the candidate location. This lets users double-check that their coordinate is reasonable.

**Comment by twentysixmoons** [comment]
How easy/hard is this to do?

**Comment by prestonp** [comment]
I don't think this should be too difficult. We would just need to display a map and a drop a marker where the lat/long coords are for a given address.

I'm thinkin the flow would be: user enters address, clicks submit then they see the page showing them the various zones they are in + a map/marker of the address.

That way they can just go "hm that marker is no where near the address, let me double check"
